lib/kube/kubeconfig/kubeconfig.go

```diff
@@ -98,6 +100,14 @@ func Update(path string, v Values) error {
 	if v.Exec != nil {
 		// Called from tsh, use the exec plugin model.
 		clusterName := v.TeleportClusterName
+		var envVars []clientcmdapi.ExecEnvVar
+		if v.Exec.Env != nil {
+			envVars = make([]clientcmdapi.ExecEnvVar, 0)
```

Comment:
This line is not necessary. `append` will work on a nil slice just fine, so no reason to make a size 0 slice first.